aurelia-slickgrid/src/aurelia-slickgrid/editors/singleSelectEditor.ts

```diff
@@ -38,13 +39,20 @@ export class SingleSelectEditor implements Editor {
    * The property name for labels in the collection
    */
   labelName: string;
+  /**
+   * The i18n aurelia library
+   */
+  private _i18n: I18N;
 
   constructor(private args: any) {
+    this._i18n = this.args.column.params.i18n;
```

Comment:
same as before.. this will probably fail at Build, `args` could possibly be undefined here